hack/update-codegen.sh

```diff
@@ -47,5 +47,12 @@ ${CONTROLLER_GEN} \
     crd \
     rbac:roleName=manager-role \
     webhook \
-    paths="./test/e2e/reconciler/cluster/..." \
-    output:crd:artifacts:config=test/e2e/reconciler/cluster/
+    paths="./test/e2e/fixtures/wildwest/..." \
+    output:crd:artifacts:config=test/e2e/fixtures/wildwest/
```

Comment:
Unrelated but this seems to have been broken for a long time, just nobody changed the test type.